akka-http-core/src/main/java/akka/http/javadsl/model/headers/EntityTagRanges.java

```diff
@@ -0,0 +1,20 @@
+/**
```

Comment:
Argh, the other PR got mixed up in this one. I must have messed up stashing, will fix.